scripts/release.sh

```diff
@@ -25,11 +25,19 @@ else
 fi
 
 echo ""
-echo "Starting Stage 1 Deploy"
+echo "Preparing Scala-Pact for publishing"
 
-bash scripts/release-stage1-libs.sh
+sbt prepareScalaPactPublish
```

Comment:
What exactly is happening here? Will this `publishSigned` every sub-project to the remote staging repo? Then when we do the `sonatypeBundleRelease` pull them down locally, sign them, then release in one upload? Or can we do the whole thing locally until the signed artefact upload

Reply:
No it's doing what you want it to do.
If you're artefacts are marked as SNAPSHOT then the publishSigned process uploads them to sonatype one by boring one as normal.
For non-SNAPSHOT artefacts, the signed artefacts are gathered in `<root>/target/sonatype-staging/` locally on your machine, and then bulk uploaded.